Tests/TestingTests/EventRecorderTests.swift

```diff
@@ -202,6 +207,33 @@ struct EventRecorderTests {
     #expect(match.output.2 == 4)
   }
 #endif
+
+  @Test("JUnitXMLRecorder outputs valid XML")
+  func junitXMLIsValid() async throws {
```

Comment:
Feel like we ought to split the ConsoleOutput and JUnit recorder types into their own test files & types, since I could imagine the new JUnit one gaining more tests over time

Reply:
Indeed, but I'll save that for when we write those tests.